query-coordinator/src/main/scala/com/socrata/querycoordinator/resources/QueryService.scala

```diff
@@ -61,6 +62,9 @@ trait QueryService {
      case NoSuchColumn(column, _) =>
        (noSuchColumn, s"No such column: ${column.name}; position: $position",
         Map(qpColumn -> JString(column.name)), position)
+     case NoSuchTable(table, _) =>
```

Comment:
Where did we add `NoSuchTable` SoQLException?

Reply:
Over in soql-reference: https://github.com/socrata-platform/soql-reference/pull/130